src/Integration.Vsix/SonarLintTagger/IssueTagger.cs

```diff
@@ -25,20 +25,27 @@
 using Microsoft.VisualStudio.Text.Adornments;
 using Microsoft.VisualStudio.Text.Tagging;
 using Sonarlint;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Utilities;
+using EnvDTE;
+using System.Diagnostics;
 
 namespace SonarLint.VisualStudio.Integration.Vsix
 {
     ///<summary>
-    /// Tracks Sonar errors for a specific buffer.
+    /// Tracks SonarLint errors for a specific buffer.
     ///</summary>
     /// <remarks><para>The lifespan of this object is tied to the lifespan of the taggers on the view. On creation of the first tagger,
     /// it starts tracking errors. On the disposal of the last tagger, it shuts down.</para>
     /// </remarks>
-    internal class IssueTracker : ITagger<IErrorTag>, IDisposable
+    internal class IssueTagger : ITagger<IErrorTag>, IDisposable
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix this implementation of 'IDisposable' to conform to the dispose pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3881)